src/renderer/modules/webpack/common/index.ts

```diff
@@ -0,0 +1,63 @@
+import { ModuleExports } from "../../../../types";
+import { error } from "../../logger";
+/**/
+function importTimeout<T extends ModuleExports>(
+  name: string,
+  moduleImport: Promise<T>,
+  cb: (mod: T) => void,
+): void {
+  const timeout = setTimeout(
+    () => error("Replugged", "CommonModules", void 0, `Could not find module "${name}"`),
+    5_000,
+  );
+  void moduleImport.then((mod) => {
+    clearTimeout(timeout);
+    cb(mod);
+  });
+}
```

Comment:
I think we should actually stop the queries, not just do this

Reply:
My objection to stopping the query is mostly theoretical given the speed of my computer: if for some reason a module takes longer than five seconds to be found, but eventually becomes available, then stopping the query will make the erroneous assertion that the filter is wrong and will prevent there from being any information to the contrary (i.e. the presence of the module in `replugged.common` despite the error message). Besides, the performance impact of this for the limited number of common modules is negligible at worst.